package.json

```diff
@@ -35,6 +35,7 @@
     "@types/morgan": "^1.9.10",
     "@types/node": "^20.8.0",
     "@vitest/coverage-v8": "^3.2.4",
+    "drizzle-kit": "^0.31.5",
```

Comment:
would be good to set up some new script for it too, so that you can do `npm run migrate` etc